### PR TITLE
Support several multipliers for a QSO

### LINKED ISSUE #411: Submitting contact with several multipliers
California QSO party has this concept of county lines. Contact with county line stations gives two multipliers.
Are there any ways for recording several multipliers in one QSO?

### PR DESCRIPTION
**Pull request by zcsahok**
Allow generating several multipliers for a single QSO. These are generated by the corresponding contest plugin as a space separated list. Usages include both explicit multipliers as for California QSO party and also derived ones as for EU-DX. In the latter case both DXCC and the EU region code count as multipliers.

Added rules for EU-DX.

Resolves #411.

### CODE REVIEW

File: src/addmult.c [hunk 1]
```diff
@@ -39,6 +39,49 @@
 
 GPtrArray *mults_possible;
```
Comment (dl1jbe) [inline src/addmult.c:41]: Maybe locate function near related remember_multi()?

**Review by dl1jbe — APPROVED** [review]
Sorry for missing that one before.

The solution will certainly work for now. Would be nice if the single mults were vertically aligned, but that looks difficult in the actual case.

In the long run a solution with a separate handling of the different multipliers would  be desirable. (Same goes for the separate scoring of multipliers per txmode.)

Otherwise LGTM.

**Comment by dl1jbe** [comment]
Btw, we can give us some more room for multipliers if we change the 'displayed' date from 12-Aug-24 to 12Aug24. 

Similar, but not as readable would be to drop the ':' from 17:09. In summary that are 2..3 characters more for the information on the right side.

The displayed line would than read as

` 40CW  20Feb22 1709 0004  VE3KL          599  599  23            VE3         1  `

Be aware that I am NOT talking about changing the line format in log file, but only on display. Downside: We need separate functions to format both (But that may not be a bad decision.).

**Comment by zcsahok** [comment]
Yes, the on-screen format could be tuned. We still have to store something in the log file, so there it will be truncated. As this is quite a special case, I'd postpone any changes for later.

**Pull request merged**